docs/src/main/sphinx/connector/hudi.md

```diff
@@ -12,8 +12,8 @@ To use the Hudi connector, you need:
 
 - Hudi version 0.12.3 or higher.
 - Network access from the Trino coordinator and workers to the Hudi storage.
-- Access to a Hive metastore service (HMS).
-- Network access from the Trino coordinator to the HMS.
+- Access to a Hive metastore service (HMS) or AWS Glue.
```

Comment:
> Update hudi.md 

The commit title is too generic. You can remove the commit body and mentioned the change in a title instead.

Reply:
sounds good.